tests/v2/validation/workloads/workload_test.go

```diff
@@ -27,7 +27,9 @@ import (
 )
 
 const (
-	imageName = "nginx"
+	imageName       = "nginx"
```

Comment:
I know this isn't part of this PR, but can you update this constant to be similar to the other images "nginxImageName"

Reply:
I can